<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogo_InsercionDatosBD</class>
 <widget class="QDialog" name="Dialogo_InsercionDatosBD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insercion datos BD Sqlite</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_nombre_base_datos">
      <property name="text">
       <string>Nombre base de datos:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_nombre_base_datos"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_conectar">
      <property name="text">
       <string>Conectar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>23</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbl_nombre_tabla">
      <property name="text">
       <string>Nombre tabla</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txt_nombre_tabla"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbl_id">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txt_id"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lbl_nombre">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txt_nombre"/>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btn_insertar_registro">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Insertar Registro</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
